lib/librato/rack/validating_queue.rb

```diff
@@ -6,20 +6,21 @@ class Rack
     #
     class ValidatingQueue < Librato::Metrics::Queue
       METRIC_NAME_REGEX = /\A[-.:_\w]{1,255}\z/
-      SOURCE_NAME_REGEX = /\A[-:A-Za-z0-9_.]{1,255}\z/
+      TAGS_KEY_REGEX = /\A[-.:_\w]{1,64}\z/
+      TAGS_VALUE_REGEX = /\A[-.:_\w]{1,256}\z/
 
       attr_accessor :logger
 
       # screen all measurements for validity before sending
       def submit
-        @queued[:gauges].delete_if do |entry|
+        @queued[:measurements].delete_if do |entry|
```

Comment:
> @queued[:metrics] here?

Following up on our conversation, decided to keep this as `measurements` for consistency w/ API top-level key: https://www.librato.com/docs/api/#create-a-measurement